stixcore/processing/pipeline.py

```diff
@@ -109,6 +116,9 @@ def __init__(self, tm_file):
         logging.StreamHandler.__init__(self)
 
         self.tm_file = tm_file
+
+        PipelineErrorReport.CURRENT_TM = (tm_file, datetime.now())
```

Comment:
Not clear why referencing the class there rather then `self.CURRENT_TM` also caps are usually denotes static content